scene/3d/skeleton.cpp

```diff
@@ -929,6 +934,10 @@ void Skeleton::_bind_methods() {
 	ADD_PROPERTY(PropertyInfo(Variant::BOOL, "animate_physical_bones"), "set_animate_physical_bones", "get_animate_physical_bones");
 #endif // _3D_DISABLED
 
+#ifdef TOOLS_ENABLED
+	ADD_SIGNAL(MethodInfo("pose_updated"));
+#endif // TOOLS_ENABLED
```

Comment:
That doesn't seem right. It's still unclear to me how to properly propagate info from scene to editor, but we don't use signals for that usually. Either change receptors or `change_notify`. @reduz would know more.